### ISSUE TITLE
Need a README File

### ISSUE THREAD

**Issue by Rasesh2005** [Rasesh2005/Video-Downloader-Website#3]
Require a clean README.md file

**Comment by AvineshTripathi** [comment]
Can I take this up ?

**Comment by SunnyBhagwat35** [comment]
Hi, I'm a beginner at contributing to open source.
I would like to start with this if it's Ok.
Thank you and best regards

**Comment by Rasesh2005** [comment]
I need a readme file that demonstrates using the website along with running the program on localhost.

**Comment by AvineshTripathi** [comment]
I can do that

**Comment by AgentA07** [comment]
Can i take this up?

**Comment by AgentA07** [comment]
I will make a detailed README file

**Comment by Rasesh2005** [comment]
Whoever Wants to take this up i am assigning them , I will look into every Pull Request and Merge the best one and put a "hacktober-accepted" tag on rest of them, for the effort you guys will put in making a contribution ;)

**Comment by SunnyBhagwat35** [comment]
Hey, can I take this up too?

**Comment by Rasesh2005** [comment]
yeah I will Assign you too ;)

**Comment by Rasesh2005** [comment]
REMINDER: Your README File Should not be a 2 to 5 liner information, It needs photos(screenshots of how to use the website) and some css or coloring to make it a bit attractive

**Comment by chirantan-basu01** [comment]
Can I fix some issues in your repo?

**Comment by NuclearCactus** [comment]
Can I please help you by taking up this issue? I'm an experienced technical content writer and can help you create a clean README file.

**Comment by Rasesh2005** [comment]
@NuclearCactus , Yes You may. I am assigning you also for the resolution of issue

**Comment by Rasesh2005** [comment]
For Your Information This issue is still OPEN if anyone wants to participate

**Comment by AvineshTripathi** [comment]
@Rasesh2005 I was about to give a PR for the same readme with same template

**Comment by Rasesh2005** [comment]
If anyone wants to try making a better on they can still try
